app.js

```diff
@@ -1,4 +1,8 @@
-const button = document.getElementById("button");
-button.addEventListener("click", () => {
+const buttton = document.getElementById("buttton");
+buttton.addEventListener("click", () => {
```

Comment:
There's a typo, but that's just a training. In real project I would request to also fix that. ;)